.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2E Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  e2e-tests:
+    name: Run E2E tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+
+    - name: Install system dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y chromium
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt
+        pip install -r process_report/tests/test-requirements.txt
+
+    - name: Run E2E Tests
+      run: |
+        pytest process_report/tests/e2e/test_e2e_pipeline.py -v
```

Comment:
Just a suggestion, in my opinion we should run this with `-s` flag so we can see the output when the tests run.